contracts/prize-strategy/multiple-winners/MultipleWinners.sol

```diff
@@ -61,9 +68,69 @@ contract MultipleWinners is PeriodicPrizeStrategy {
     return __numberOfWinners;
   }
 
+  function setPrizeSplit(MultipleWinnersPrizeSplit[2] memory prizeStrategySplit) external onlyOwner {
+    for (uint256 index = 0; index < prizeStrategySplit.length; index++) {
+        MultipleWinnersPrizeSplit memory split = prizeStrategySplit[index];
+
+        // If MultipleWinnersPrizeSplit is non-zero address set the prize split configuation.
+        if (split.target != address(0)) {
+            // Split percentage must be below 1000 and greater then 0 (e.x. 200 is equal to 20% percent)
+            // The range from 0 to 1000 is used for single decimal precision (e.x. 15 is 1.5%)
+            require(
+                split.percentage > 0 && split.percentage < 1000,
+                "MultipleWinners:invalid-prizesplit-percentage-amount"
+            );
+
+            _prizeSplits[index] = split;
+        }
+    }
```

Comment:
Add an event so we can track state changes externally

Reply:
👍

Reply:
Added an event for when the prize split is distributed too.